spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -63,4 +63,7 @@ private SparkSQLProperties() {}
   // Controls the WAP branch used for write-audit-publish workflow.
   // When set, new snapshots will be committed to this branch.
   public static final String WAP_BRANCH = "spark.wap.branch";
+
+  // Controls read split size for the individual table
+  public static final String TEMPLATED_SPLIT_SIZE = "spark.sql.iceberg.%s.read-split-size";
```

Comment:
I wonder whether we should offer a generic templated version to override table properties rather than adding a SQL  property for each table property. Let us discuss this during the community sync tomorrow.

Reply:
Also, will we be able to handle quoted identifiers?

Reply:
@aokolnychyi Hi, may I know what's the outcome of the community sync discussion?